2019Labs/CustomSecurityContent/documentation/lab3_profiles.adoc

```diff
@@ -43,25 +45,36 @@ image::0-01-setup.png[]
 
 == Creating a new empty profile
 
-The basic building block of a security policy is ComplianceAsCode is a rule. The rule represents a single configuration setting, eg. "Password length is at least 8 characters" or "Logging is enabled".
+The basic building block of a security policy is ComplianceAsCode is a rule.
+The rule represents a single configuration setting, eg.
+"Password length is at least 8 characters" or "Logging is enabled".
 
-A set of rules is called a profile. A profile represents a specific security policy. In ComplianceAsCode we have multiple profiles.
+A set of rules is called a profile.
+A profile represents a specific security policy.
+In ComplianceAsCode we have multiple profiles.
 
-We have policies for different products. By using term "product" we mean operating systems or applications, eg. Red Hat Enterprise Linux 8, OpenShift Container Platform 3.
-The products are represented by directories in the root of the ComplianceAsCode repository, eg. `rhel7`, `rhel8`, `ocp3` directories.
+We have policies for different products.
```

Comment:
I suggest to say `profiles` instead of `policies`, to link with previous sentence.